tests/ui/push-health/TestFailure_test.jsx

```diff
@@ -77,28 +58,25 @@ describe('TestFailure', () => {
   });
 
   test('should not show details by default', async () => {
-    const { container, getByText } = render(testTestFailure(testFailure));
-    useStyles(container);
-
-    // Must use .toBeVisible() rather than .toBeInTheDocument because
-    // Collapse just hides elements, doesn't remove them.
-    expect(
-      await waitForElement(() =>
-        getByText('Transactions that explicitly commit ', {
-          exact: false,
-        }),
-      ),
-    ).not.toBeVisible();
+    const { getByText, getByTestId } = render(testTestFailure(testFailure));
+    const logLineToggle = await waitForElement(() => getByTestId('log-lines'));
+
+    // For collapsible components, you must check for 'collapse' (hidden) or 'collapse show' (visible)
```

Comment:
Ahh, heh.  Good call.  :)  Yeah, this is a good way to handle this.  I was a bit heavy-handed with my approach....